### PR TITLE
Ports In hand bola sprites from TG

### PR DESCRIPTION
**Pull request by Bm0n**
## What Does This PR Do
Adds in hand bola sprites for all four bola types from TG Station.

Original PR from TG: https://github.com/tgstation/tgstation/pull/49606
## Why It's Good For The Game
We currently do not have an in hand sprite for bolas. This is a problem since bolas are a weapon and not being able to see one coming can mean life or death.

## Images of changes
bola
## Changelog
:cl: Bmon
tweak: Ported in hand bola sprites from TG
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/legcuffs.dm [hunk 1]
```diff
@@ -151,6 +151,9 @@
 	name = "bola"
 	desc = "A restraining device designed to be thrown at the target. Upon connecting with said target, it will wrap around their legs, making it difficult for them to move quickly."
 	icon_state = "bola"
+	item_state = "bola"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
```
Comment (hal9000PR) [inline code/game/objects/items/weapons/legcuffs.dm:156]: this is inherited from `/obj/item` so its not needed.

**Review by hal9000PR — APPROVED** [review]
looks good otherwise

File: code/game/objects/items/weapons/legcuffs.dm (hunk 1 above)
Comment (Bm0n) [reply code/game/objects/items/weapons/legcuffs.dm]: Corrected this, should be good now!

**Review by SteelSlayer — APPROVED** [review]

**Comment by TORE01** [comment]
8206af752ef52f045083ecc8de56617d

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**